contracts/ExitModule.sol

```diff
@@ -25,12 +25,14 @@ contract Exit is Module {
     constructor(
         address _owner,
         address _avatar,
+        address _target,
```

Comment:
we should add the description in natspec for `target`

Reply:
Yes, we should do both. On it.